src/main/java/edu/ucsb/cs156/courses/collections/ConvertedSectionCollection.java

```diff
@@ -23,9 +23,8 @@ List<ConvertedSection> findByQuarterRangeAndInstructor(
       String startQuarter, String endQuarter, String instructor, String functionCode);
 
   @Query(
-      "{'courseInfo.quarter': { $gte: ?0, $lte: ?1 }, 'section.timeLocations.building': { $regex: ?2, $options: 'i' } }")
-  List<ConvertedSection> findByQuarterRangeAndBuildingCode(
-      String startQuarter, String endQuarter, String buildingCode);
+      "{'courseInfo.quarter': { $eq: ?0 }, 'section.timeLocations.building': { $regex: ?1, $options: 'i' } }")
+  List<ConvertedSection> findByQuarterAndBuildingCode(String quarter, String buildingCode);
```

Comment:
Unfortunately, this breaks the existing search by building page.

What if you left the old query in place, but added the new one *in addition* to the old one, at least until you:

* Follow this PR up with one that changes the frontend to match the new backend endpoints
*  THEN remove the old backend endpoints?

You want to always leave the app in a working state, and this breaks existing code.